pages/Documents/ConversationalAI/BotAnalytics/overview.md

```diff
@@ -19,5 +19,5 @@ The Bot Analytics application is intended to provide real time data on your bots
 
 **To access the Bot Analytics application**
 
-1. On the left sidebar in Conversational Cloud, click the <img class="inlineimage" style="width:30px" src="img/ConvoBuilder/icon_cb.png"> icon.
+1. On the left sidebar in Conversational Cloud, click the <img class="inlineimage" style="width:30px" src="img/ConvoBuilder/icon_cb.png" alt="Bot icon"> icon.
```

Comment:
(Side note: It’s not good practice to say something is an icon. For a screen reader user, saying what bot this is is really the only thing that would help—what bot do they need to click? That could be the “alt” text: “x bot".)